### PR TITLE
Stock market map

### PR DESCRIPTION
**Pull request by mumble-07**
### **THE STORY**

> This PR is for mapping of IEX stock market data and rendering it to views. market index page.
> 5000 stock symbols were fetched via IEX ruby client; alphabetically. 
> A market controller was generated
> Methods for conversion of stock price from usd to php were added in MarketsHelper
>  +5% for buy
> -5% for sell
> The current usd to php conversion used is P50.06, to changed the value edit it in MarketsHelper
### **SCREENSHOTS**

image

### CODE REVIEW

File: app/helpers/markets_helper.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    convert_price = iex_price * usd_to_php
+    with_interest_total = convert_price + (convert_price * 0.05)
+    number_to_currency(with_interest_total, unit: 'PHP ')
+  end
+
+  def current_sell_price(iex_price)
+    usd_to_php = 50.06
+    convert_price = iex_price * usd_to_php
+    with_interest_total = convert_price - (convert_price * 0.05)
+    number_to_currency(with_interest_total, unit: 'PHP ')
```
Comment (michael-dollosa) [inline app/helpers/markets_helper.rb:13]: you can improve these helpers right away to stay DRY

example, I can just right a method that accepts a boolean value (true for buy, false for sell) and returns the total that will be used by number_to_currency rails helper

```
def calculate_total(bool)
    usd_to_php = 50.06
    convert_price = iex_price * usd_to_php
    with_interest_total = bool ? convert_price + (convert_price * 0.05) : convert_price - (convert_price * 0.05)
end
```

**Review by michael-dollosa — CHANGES_REQUESTED** [review]
you can improve these helpers right away to stay DRY

example, I can just right a method that accepts a boolean value (true for buy, false for sell) and returns the total that will be used by number_to_currency rails helper

```
def calculate_total(bool)
    usd_to_php = 50.06
    convert_price = iex_price * usd_to_php
    with_interest_total = bool ? convert_price + (convert_price * 0.05) : convert_price - (convert_price * 0.05)
end
```

File: app/helpers/markets_helper.rb (hunk 1 above)
Comment (mumble-07) [reply app/helpers/markets_helper.rb]: Hello, oh yeah right! That's a good idea, thanks! 

Please check if this is okay: https://github.com/mumble-07/emporia-wallet-app/pull/13/commits/bbd5d4beec58f4f7d8eca2551063ac7b505e0b31

**Review by michael-dollosa — APPROVED** [review]
lgtm

**Pull request merged**